bin/crypt.py

```diff
@@ -13,23 +13,24 @@
   -k KEY, --key KEY  Encrypt/Decrypt Key.
   -d, --decrypt      Flag to decrypt.
 '''
-import os
 import argparse
 import base64
+import os
 
+_stash = globals()['_stash']
```

Comment:
I'm fairly certain this does nothing - `globals()` is always the current global namespace, which we are in right now, so this is equivalent to `_stash = _stash`.

Reply:
There were existing instances of this elsewhere in the code so I replicated it because it makes the assumptions explicit to the reader.  Would __from stash import \__stash__ work as alternative?

Reply:
It indeed does nothing. I added it mainly to keep the IDE happy.

Reply:
;-)  IDE probably runs flake8 or some other linter under the covers.